packages/connect-iframe/webpack/utils.ts

```diff
@@ -0,0 +1,30 @@
+import path from 'path';
+
+type Project = 'iframe' | 'popup' | 'suite-web';
+
+export const getDistPathForProject = (project: Project = 'iframe') => {
+    const basePath = path.join(__dirname, '..', '..');
+    switch (project) {
+        case 'iframe':
+            return path.join(basePath, 'connect-iframe', 'build');
+        case 'suite-web':
+            return path.join(basePath, 'suite-web', 'build');
+        case 'popup':
+            return path.join(basePath, 'connect-popup', 'build');
+        default:
+            throw new Error('Missing project.');
+    }
+};
+
+// TODO: maybe sharedworker is not needed anymore.
```

Comment:
yes, maybe not. it should only be build by connect and suite is using it via this clumsy mechanism here https://github.com/trezor/trezor-suite/blob/feat/use-core-in-suite-web/suite-common/connect-init/src/connectInitThunks.ts#L111

Reply:
If I understand well with the latest changes it is still needed for developer/testing environments  in:
* connect-explorer - https://github.com/trezor/trezor-suite/blob/develop/suite-common/connect-init/src/connectInitThunks.ts#L136
* suite-web - https://github.com/trezor/trezor-suite/blob/develop/suite-common/connect-init/src/connectInitThunks.ts#L136

Reply:
yes, but there is only one "source of build". previously suite-web and connect had their own builds which was wrong. Now only connect builds and uses its instance of sharedworker as a default. 

This logic in connectInitThunks is there because suite has its own copy of connect but we do not want this copy to by using also a custom copy of sharedworker - it should be looking for the "master" one.

Reply:
How did you get to that error? 
I can see it is here `https://dev.suite.sldev.cz/suite-web/feat/use-core-in-suite-web/web/workers/sessions-background-sharedworker.js`  I can see the worker is there but the request you displayed in the image is missing `web` in `https://dev.suite.sldev.cz/suite-web/feat/use-core-in-suite-web/web/` so I am curious how it happened.

Reply:
Fixed by https://github.com/trezor/trezor-suite/pull/14959/commits/bb3eb4d186f45c783eeb7f932f232214e19864a5

Reply:
I don't think it works. I am still getting the same error on connect-explorer build on sldev. did it build the latest commit?

Reply:
It was missing for connect-explorer build. This https://github.com/trezor/trezor-suite/pull/14959/commits/7404ffa1a591e2b403ce5b4477cb61ccbb6f86b1fixes it.